charts/ccsm-helm/charts/tasklist/values.yaml

```diff
@@ -2,34 +2,3 @@ global:
   image:
     # global.image.repository overwrites the default repository to the operate image
     repository: camunda/tasklist
-
-logging:
-  level:
-    ROOT: INFO
-    org.camunda.tasklist: DEBUG
```

Comment:
:question: Is there no logging configuration for tasklist anymore?

Reply:
This was never used in the chart :shrug: I have to check how to configure it probably similar to #148